code/game/objects/items/shields.dm

```diff
@@ -357,20 +358,47 @@ obj/item/shield/riot/bullet_proof
 	block_chance = 50
 
 /obj/item/shield/riot/implant
-	name = "riot tower shield"
-	desc = "A massive shield that can block a lot of attacks and can take a lot of abuse before breaking." //It cant break unless it is removed from the implant
+	name = "telescoping shield implant"
+	desc = "A compact, arm-mounted telescopic shield. While nigh-indestructible when powered by a host user, it will eventually overload from damage. Recharges while inside its implant."
 	item_state = "metal"
 	icon_state = "metal"
-	block_chance = 30 //May be big but hard to move around to block.
+	block_chance = 50
 	slowdown = 1
```

Comment:
your also dealing with slowdown, when you have it out for that reason too. Its balanced around the "I can block just a bit better ranged well creeping to the baddie" but in melee the antag/graytide can still use toolbox/esword and not get meme clucked by a sec officer with a shield that blocks 50%of melee.

Reply:
Consider getting a real shield then.